src/main/java/org/jsw/helpers/PriceList.java

```diff
@@ -0,0 +1,48 @@
+package org.jsw.helpers;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class PriceList {
+	public void create(String name, List<String> productType, String fileName) throws IOException, JSONException {
+		JSONObject order = new JSONObject();	
+		int price;
+		int min = 5;
+		int max = 10;
+		
+		try {
+			order.put("Bakery Name", name);
+			
+			JSONObject product = new JSONObject();
+			
+			for(String type : productType ) {
+				price = ThreadLocalRandom.current().nextInt(min, max + 1);
```

Comment:
The scenario files have already specified the prices for the different bakeries, this is why you should get it from the agents directly!